Makefile

```
@@ -99,10 +101,22 @@ livedoc:
 	$(MAKE) doc
 	python3 -m http.server 8888 --directory doc/_build
 
-package := atd
+# Prepare the opam files for a release. They're derived from 'dune-project'
+# by 'dune build' and then patched to work around some issues.
+# https://discuss.ocaml.org/t/when-exactly-does-dune-generate-opam-files/8292/2
+#
+.PHONY: opam-files
+opam-files:
+	$(DUNE) build
```

```suggestion
	$(DUNE) build *.opam
```
 would be a bit faster, there's no need to build all the code to generate the opam files. But that's nitpicking

---

that's great to know, thank you.